docs/en/sql-reference/functions/ext-dict-functions.md

```diff
@@ -481,44 +443,1954 @@ SELECT dictGetAll('regexp_dict', 'tag', 'foobarbaz', 2);
 └──────────────────────────────────────────────────┘
 ```
 
-## Other Functions
+## dictGetInt8
+
+Retrieves type Int8 values from a dictionary.
 
-ClickHouse supports specialized functions that convert dictionary attribute values to a specific data type regardless of the dictionary configuration.
+**Syntax**
 
-Functions:
+```sql
+dictGetInt8('dictionary_name', 'attribute_name', key)
+```
 
-- `dictGetInt8`, `dictGetInt16`, `dictGetInt32`, `dictGetInt64`
-- `dictGetUInt8`, `dictGetUInt16`, `dictGetUInt32`, `dictGetUInt64`
-- `dictGetFloat32`, `dictGetFloat64`
-- `dictGetDate`
-- `dictGetDateTime`
-- `dictGetUUID`
-- `dictGetString`
-- `dictGetIPv4`, `dictGetIPv6`
+**Arguments**
 
-All these functions have the `OrDefault` modification. For example, `dictGetDateOrDefault`.
+- `dictionary_name`: the name of the dictionary you want to retrieve data from. [String literal](../syntax#syntax-string-literal)
+- `attribute_name`: the attribute whose value you want to get. [String literal](../syntax#syntax-string-literal), or [tuple](../data-types/tuple) of column names.
+- `key`: the key associated with the attribute. [Expression](../syntax#syntax-expressions) returning dictionary key-type value.
 
-Syntax:
+**Returned value**
 
-``` sql
-dictGet[Type]('dict_name', 'attr_name', id_expr)
-dictGet[Type]OrDefault('dict_name', 'attr_name', id_expr, default_value_expr)
+The value of the dictionary attribute that corresponds to the given key. This value is parsed and returned as an `Int8` type.
+
+**Implementation details**
+
+This function will throw an exception if the key does not exist in the dictionary.
```

Comment:
This is not true, this function will throw an exception if it cannot parse the value of the attribute or the value does not match the attribute data type, not if the key does not exist.